src/engine/schema.rs

```diff
@@ -2650,7 +2851,7 @@ mod tests {
         /*
             input ProjectInput {
                 $EXISTING: ProjectQueryInput
-                $NEW: ProjectCreateMutationInput
+                NEW: ProjectCreateMutationInput
```

Comment:
Did you mean to change this here, or did it get inadvertently pulled in from the other PR to back out the `$` prefixes from the schema?

Reply:
Likely accidental change as I was switching branches. All of it will be corrected in the `$` PR which is already on deck.